src/components/Badge/Badge.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
```

Comment:
This file can be removed, if there are no special cases to check here (other than `should render`)

Reply:
removed